x-element.js

```diff
@@ -899,110 +975,122 @@ export default class XElement extends HTMLElement {
   static #prototypeInterface = new Set(Object.getOwnPropertyNames(XElement.prototype));
 }
 
-/** Wrapper to document public interface to the default templating engine. */
-class TemplateEngine {
+// TODO: If / when we remove `repeat` in favor of `map` — we can probably delete
```

Comment:
The only thing we really need to do is have the next release be a `2.x` release so that we’re following semantic versioning. It’s not much of a hassle to drag along `repeat` until we’re actually ready to make that change though 👌 

_This way_, we can continue shipping incremental changes in `1.x` without having to manage multiple, major versions or anything.